### PR TITLE
chore: Patch 4.22.6

### PR DESCRIPTION
**Pull request by andreas-unleash**
Includes:
- Stickiness options from context fields. 
- Docs update about project mode and project stickiness
## About the changes

Closes #

### CODE REVIEW

File: frontend/src/component/feature/StrategyTypes/FlexibleStrategy/StickinessSelect/StickinessSelect.tsx [hunk 1]
```diff
@@ -25,20 +24,27 @@ export const StickinessSelect = ({
...
-                    contextDefinition =>
-                        !builtInStickinessOptions.find(
-                            builtInStickinessOption =>
-                                builtInStickinessOption.key ===
-                                contextDefinition.name
-                        )
-                )
-                .map(c => ({ key: c.name, label: c.name }))
-        );
+    const resolveStickinessOptions = () => {
```
Comment (kwasniew) [inline frontend/src/component/feature/StrategyTypes/FlexibleStrategy/StickinessSelect/StickinessSelect.tsx:27]: We started adding more tests around new behaviors recently. This one should be easy to test using this approach: https://github.com/Unleash/unleash/blob/239e338daa430cca2ade8dbe2e61883e698b3b5e/frontend/src/component/changeRequest/ChangeRequestSidebar/EnvironmentChangeRequest/EnvironmentChangeRequestTitle.test.tsx

**Review by kwasniew — APPROVED** [review]

File: frontend/src/component/feature/StrategyTypes/FlexibleStrategy/StickinessSelect/StickinessSelect.tsx (hunk 1 above)
Comment (andreas-unleash) [reply frontend/src/component/feature/StrategyTypes/FlexibleStrategy/StickinessSelect/StickinessSelect.tsx:27]: Thanks, but I am cherry picking commits, not implementing anything here

Comment (kwasniew) [reply frontend/src/component/feature/StrategyTypes/FlexibleStrategy/StickinessSelect/StickinessSelect.tsx:27]: Yeah I noticed after leaving the comment :)

Comment (andreas-unleash) [reply frontend/src/component/feature/StrategyTypes/FlexibleStrategy/StickinessSelect/StickinessSelect.tsx:27]: Cherry picked the BE protection for empty string stickiness as well

**Pull request merged**